dashboard/app/models/pd/application/email.rb

```diff
@@ -34,7 +34,20 @@ def mark_sent!
     end
 
     def self.send_all_queued_emails
-      unsent.find_each(&:send!)
+      errors = {}
+      unsent.find_each do |email|
+        email.send!
+      rescue => e
+        errors[email.id] = "#{e.message}, #{e.backtrace.first}"
+      end
+
+      if errors.any?
+        msg = "Error sending emails for applications. Errors:\n"
+        errors.each do |application_id, error|
+          msg << "Application #{application_id}: #{error}"
+        end
+        raise msg
```

Comment:
Does HoneyBadger cope okay with an array of messages?

Reply:
good question - this is actually one long string, not an array, but I added newlines to make it easier to read if there are multiple